spec/javascripts/components/single-page-notification-button-spec.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-env jasmine, jquery */
```

Comment:
Nitpick: is `jquery` needed here? I can't see any in this file.

Reply:
Nope, I let this in from earlier iterations of this file, good spot